tests/Feature/TrafficAnalytics/SentinelTrafficCacheTest.php

```
@@ -0,0 +1,31 @@
+use App\Models\Team;
+use App\Services\SentinelTrafficClient;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Cache;
+
+uses(RefreshDatabase::class);
+
+class CountingTrafficClient extends SentinelTrafficClient
+{
+    public int $calls = 0;
+
+    protected function remoteFetch(string $url): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$url'. (undefined)

(UnusedFormalParameter)